src/backend/app/db/models.py

```diff
@@ -221,6 +221,41 @@ async def all(
             )
             return await cur.fetchall()
 
+    @classmethod
+    async def delete(cls, db: Connection, user_id: int) -> bool:
+        """Delete a user and their related data."""
+        async with db.cursor() as cur:
```

Comment:
Can we change this to a single SQL and await the execution at once? What do you think?

Reply:
Unfortunately that's not possible with psycopg! It's only one statement per cursor 😅
(SQLAlchemy hides this fact!).

It looks inefficient, but it still executes them all in the same session, so it's not bad

Reply:
I see you have done a similar delete api for the organisation with a single sql.

Reply:
Oh right! Does it actually work? I would think it might error, but could be wrong. 

I think the project deletion has it separated for this reason

Reply:
Ah ignore me, I see what you mean now!

The org deletion is a single prepared statement, so that works fine

Reply:
I am not sure if it works, I haven't tested it yet. :thinking:

Reply:
> I am not sure if it works, I haven't tested it yet. :thinking: 

I think it does!